Providers/Modules/Plugins/Common/plugin/oms_configuration.rb

```diff
@@ -37,6 +38,11 @@ def test_onboard_file(file_name)
       end
 
       def get_proxy_config(proxy_conf_path)
+        old_proxy_conf_path = '/etc/opt/microsoft/omsagent/conf/proxy.conf'
```

Comment:
@lagalbra  This part of change is not related to diagnostic logging. These are related to your change "Ensure that proxy.conf location is backwards-compatible" which have a mismatch between the DSC files and the files in OMS-Agent-for-Linux. Should this be allowed to go or should I remove them?